lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    fursa_of_peronne_abbot: 'Saint Fursa, Abbot and Missionary',
+    gabriel_taurin_dufresse_bishop: 'Saint Gabriel-Taurin Dufresse, Bishop and Martyr',
+    gall_of_switzerland_abbot: 'Saint Gall, Abbot and Missionary',
+    genevieve_of_paris_virgin: 'Saint Genevieve, Virgin',
+    george_matulaitis_bishop: 'Blessed George Matulaitis, Bishop',
+    george_of_lydda_martyr: 'Saint George, Martyr',
+    george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr, Patron of England',
+    george_preca_priest: 'Saint George Preca, Priest',
+    gerard_of_csanad_bishop: 'Saint Gerard of Csanád, Bishop and Martyr',
+    germain_of_paris_bishop: 'Saint Germain of Paris, Bishop',
+    germanus_of_auxerre_bishop: 'Saint Germanus of Auxerre, Bishop',
+    gertrude_of_nivelles_abbess: 'Saint Gertrude of Nivelles, Abbess',
+    gertrude_the_great_virgin: 'Saint Gertrude, Virgin',*/
+    gisela_of_hungary: 'Boldog Gizella',
+    /*gobnait_of_ballyvourney_virgin: 'Saint Gobnait, Virgin',
+    gorazd_of_moravia_and_companions: 'Saint Gorazd and Companions',
+    gotthard_of_hildesheim_bishop: 'Saint Gotthard, Bishop',
+    gratia_of_cattaro_religious: 'Blessed Gratia of Cattaro, Religious',
+    gregory_grassi_francis_fogolla_and_anthony_fantosati_bishops_and_companions_martyrs: 'Saints Gregory Grassi, Francis Fogolla and Anthony Fantosati, Bishops, and Companions, Martyrs',
+    gregory_i_the_great_pope: 'Saint Gregory the Great, Pope and Doctor of the Church',
+    gregory_of_narek_abbot: 'Saint Gregory of Narek, Abbot and Doctor of the Church',
+    gregory_vii_pope: 'Saint Gregory VII, Pope',*/
+    guardian_angel_of_portugal: 'Portugália őrangyala',
+    /*gundisalvus_garcia_martyr: 'Saint Gundisalvus Garcia, Martyr',
+    gundisalvus_of_amarante_priest: 'Blessed Gundisalvus of Amarante, Priest',
+    gundisalvus_of_lagos_priest: 'Blessed Gundisalvus of Lagos, Priest',
+    hedwig_of_poland: 'Saint Hedwig of Poland',
+    hedwig_of_silesia_religious: 'Saint Hedwig, Religious',
+    helena_of_constantinople: 'Saint Helena',
+    hemma_of_gurk: 'Saint Hemma of Gurk',
+    hemming_of_turku_bishop: 'Blessed Hemming of Turku, Bishop',
+    henry_ii_emperor: 'Saint Henry',
+    henry_ii_emperor_and_cunigunde_of_luxembourg: 'Saints Henry and Cunigunde',
+    henry_of_finland_bishop: 'Saint Henry, Bishop and Martyr',
+    henry_suso_priest: 'Blessed Henry Suso, Priest',
+    hermann_joseph_of_steinfeld_priest: 'Saint Hermann Joseph, Priest',
+    hermenegild_the_visigoths_martyr: 'Saint Hermenegild, Martyr',
+    hilary_of_poitiers_bishop: 'Saint Hilary, Bishop and Doctor of the Church',
+    hilda_of_whitby_abbess: 'Saint Hilda, Abbess',
+    hildegard_of_bingen_abbess: 'Saint Hildegard of Bingen, Abbess and Doctor of the Church',
+    holy_child_of_cebu: 'The Holy Child of Cebú',
+    holy_family_of_jesus_mary_and_joseph: 'The Holy Family of Jesus, Mary and Joseph',*/
+    holy_guardian_angels: 'A szent őrzőangyalok',
```

Comment:
According to [Direktórium 2022](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=161), there is no definite article (`a`) at the beginning of the celebration name. I have no idea if that is so in the Hungarian Roman Missal.